core/src/main/java/io/questdb/cairo/mv/MatViewRefreshJob.java

```diff
@@ -610,7 +615,7 @@ private boolean refreshDependentViewsIncremental(
                 } catch (Throwable th) {
                     // If we're here, we either couldn't obtain the WAL writer or the writer couldn't write
                     // invalid state transaction. Update the in-memory state and call it a day.
-                    LOG.error()
+                    LOG.critical()
```

Comment:
If someone drops the MV during refresh, you can get an exception here. It should not be critical.

Reply:
That's a fair point. Changed back to error log level in 43d4b4e91f6ef79786addb7392a01f1b229cedd1